helpers/utils.ts

```diff
@@ -32,9 +22,7 @@ interface ChainAddressesConfig {
 // Add all multisigs here (in case there is no NormalTimelock deployed on the chain we will fallback to this config)
 export const multisigs: ChainAddressesConfig = {
   sepolia: "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb",
-};
-
-export const wBNBAddress: ChainAddressesConfig = {
-  bscmainnet: bnbMainnetAddresses.WBNB,
-  bsctestnet: bnbTestnetAddresses.WBNB,
+  ethereum: "0x285960C5B22fD66A736C7136967A3eB15e93CC67",
+  opbnbtestnet: "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf",
+  opbnbmainnet: "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207",
 };
```

Comment:
In order to support hardhat, this can be rewritten to be something like 
```
export const getMultisigAddress = async (network: Networks) => {
  const { deployer } = await getNamedAccounts();
  switch (network) {
    case ('sepolia'): {
      return "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb"
    }
    case ('ethereum'): {
      return "0x285960C5B22fD66A736C7136967A3eB15e93CC67"
    }
    case ('opbnbtestnet'): {
      return "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf"
    }

    case ('opbnbmainnet'): {
      return "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207"
    }
    case ('hardhat'): {
      return deployer
    }
  }
};```

Reply:
I think hardhat is supported anyway because the owner of the deployed contract will be the `deployer` for `!live` networks